docs/documentation/helpcenter.md

```diff
@@ -147,6 +148,11 @@ with modals, etc. should never go in the help center documentation.
 In such cases, you may be able to fix the problem by adding text in-app,
 where the user will see it as they are interacting with the feature.
 
+## Writing style
+
+There are some general style and writing conventions that should be used
+when documenting Zulip's features.
```

Comment:
I think it would sound a little better like so:
"We have general style and writing conventions that should be used as guidance when documenting Zulip's features."

Reply:
How about:

> Below are some general style and writing conventions that should be used as guidance when documenting Zulip's features.